test/zmq_proxy.xrep-xreq.js

```diff
@@ -43,7 +43,7 @@ describe('proxy.xrep-xreq', function() {
       rep.send(msg+' bar');
     });
 
-    setTimeout(function() {
+    setTimeout(function () {
```

Comment:
I always put a space after `function`. That's my consistent rule. The Crockford way.